src/subscribe/index.ts

```diff
@@ -126,6 +126,7 @@ export class Subscription extends Emitter {
         return this.pipeline.throw(...args)
     }
 
+    // TODO should we expose this to the user as no-args method?
```

Comment:
@timoxley How about this `subscription.unsubscribe()`? Is there some cases where user would this with parameters? At least internally we can call this with an `error` parameter, if I understood correctly. (Does that error parameter have some specific type?)